static/js/addons-config/string-field.js

```diff
@@ -37,9 +37,15 @@ function StringField(schema,
   return this;
 }
 
+StringField.prototype.toFormData = function(value) {
+  return value === '' ? function() {}() : value;
```

Comment:
Why not just use `undefined` instead of the empty function? That would be much more clear.

Reply:
This deal with eslint rule "no-undefined". But I think that this is not clear, neither.
I will fix this.